pilot/pkg/model/context.go

```diff
@@ -307,14 +307,32 @@ func GetProxyConfigNamespace(proxy *Proxy) string {
 
```

Comment:
this function GetProxyConfigNamespace can go to ads.go->getProxyConfigNamespace imo.